app/controllers/iterations_controller.rb

```diff
@@ -53,7 +53,11 @@ def get_customer_feedback
         pf.iteration = iter
         pf.save
 
-        url = "http://localhost:3000/feedback/#{eng.id}/#{iter.id}"
+        if Rails.env.production?
+          url = "http://esaas-engagements.herokuapp.com/feedback/#{eng.id}/#{iter.id}"
```

Comment:
is there a way to get this info from eg `request.request_uri` or some other property so that we don't need this separate check?  (If we do need a separate check, seems better to setup something in `environments/*.rb` where a global gets set - to DRY out the knowledge of what is different between environments)

Reply:
yes this information lives in request.env["REQUEST_URI"], updated to get hostname from here